package.json

```diff
@@ -1,7 +1,7 @@
 {
     "scripts": {
-        "dev":"docker compose -f docker-compose.dev.yml up -d --build",
-        "dev:attach":"docker compose -f docker-compose.dev.yml up --build",
+        "dev":"docker compose -f docker-compose.dev.yml up -d --force-recreate --build",
```

Comment:
I'm not sure about forcing container recreation. Of course, if we recreate the container we can make sure that we are developing on a clean slate, but it will also take more time to initialise all the containers every time we need to start our development. @johannalaitila do you have an opinion on this?

Reply:
It's fine for me because I think you would not run npm run dev everytime you'd wanna start the application, instead just stop/start the container directly from Docker after generating it for the first time. At least that's what I'm used to...